lib/appraisal/appraisal_file.rb

```diff
@@ -19,11 +19,9 @@ def initialize
       @gemfile = Gemfile.new
       @gemfile.load(ENV["BUNDLE_GEMFILE"] || "Gemfile")
 
-      if File.exist? path
-        run IO.read(path)
-      else
-        raise AppraisalsNotFound
-      end
+      raise AppraisalsNotFound unless File.exist?(path)
+
+      run(File.read(path))
```

Comment:
`File.read` is safer than `IO.read` (according to whoever made the rule for RuboCop).